[DOCS] removes block-config docs for source data systems in the cloud (#7871)
great-expectations/great_expectations: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```python name="tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_gcs.py save_expectation_suite"
```

> For more details on how to configure the RuntimeBatchRequest, as well as an example of how you can load data by specifying a GCS path to a single CSV, please refer to [How to connect to data on GCS using Pandas](/docs/0.15.50/guides/connecting_to_your_data/cloud/gcs/pandas)

Reviewer: ```suggestion
To configure the RuntimeBatchRequest and learn how to load data by specifying a GCS path to a single CSV, see [How to connect to data on GCS using Pandas](/docs/0.15.50/guides/connecting_to_your_data/cloud/gcs/pandas)
```
